sym_api_client_python/clients/stream_client.py

```diff
@@ -68,7 +68,7 @@ def update_room(self, stream_id, **kwargs):
             "description": "testing update room function",
         }
 
-        update_room(stream_id, kwargs=room_obj)
```

Comment:
Is it the only place in the doc where `kwargs` needs to be replaced by `**`?

Reply:
I searched for all the client class and only found this one.